guide/popular-topics/reactions.md

```diff
@@ -140,7 +140,7 @@ if (message.content === '!fruits') {
 	Promise.all([
 		message.react('🍎'),
 		message.react('🍊'),
-		message.react('🍇')
+		message.react('🍇'),
```

Comment:
This is definitely not okay.

Reply:
Agreed. This is utterly appalling, and quite frankly, disgusting.

Reply:
![Begone, Thots](https://cdn.discordapp.com/emojis/349705600059572236.png?v=1)